.github/workflows/binaries.yml

```diff
@@ -0,0 +1,275 @@
+name: "Build binaries"
+on: ["push", "pull_request"]
```

Comment:
I was feeling some climate guilt at submitting such small pull requests, while the CI includes this heavy binary release workflow…  Do you think it makes sense to limit the binaries step to only run on the `main` branch, and only after the other tests are successful?  For the main `elixir-desktop` project I can imagine that binary releases should be tested on the PR branches (sometimes), but in this example app and for downstream forks of this repo, we can probably assume that the platform builds are going to succeed so the binary step is only valuable to actually build artifacts for download.

Reply:
Probably makes sense to run on the main branch as well as on some wildcard "ci/*" so that ci changes for the binaries built can still be tested in PRs and branches